lytical/roster.json

```diff
@@ -61,6 +61,14 @@
       "materials": [
         {
           "type": "cv"
+        },
+        {
+          "type": "video",
+          "url": "ZF1H0qftw58"
+        },
+        {
+          "type": "portfolio",
+          "url": ""
```

Comment:
Can't approve the pr, if there is no link to your portfolio :(

Reply:
Oh oops! I've added it now!
...
defo not still working at this time...